### PR TITLE
[docs][cloud] Clarify VPC address restrictions

### PR DESCRIPTION
**Pull request by ddhodge**
Clarify restictions for overlapping CIDR ranges in VPCs

@netlify /preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-vpc-intro/#private-ip-address-ranges

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-vpc-intro.md [hunk 1]
```diff
@@ -69,9 +69,9 @@ Each region in multi-region clusters must be deployed in a VPC. Depending on the
 
 | Provider | VPC setup
 | :--- | :--- |
-| AWS | You need to create a VPC in each region where the cluster is to be deployed. If you intend to connect each region to the same application, ensure that the CIDRs of the VPCs do not overlap. |
-| GCP Global | Create a single global VPC; each region is deployed in the same VPC. |
-| GCP Custom | When creating the VPC, include each region where the cluster is to be deployed; each region is deployed in the same VPC |
+| AWS | You need to create a VPC in each region where the cluster is to be deployed.<br/>To deploy a cluster into those regional VPCs, ensure that the CIDRs of the VPCs do not overlap.<br/>If you intend to peer different VPCs to the same application VPC, ensure that the CIDRs of the VPCs do not overlap. |
```
Comment (vitroth) [inline docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-vpc-intro.md:72]:
```suggestion
| AWS | You need to create a VPC in each region where the cluster is to be deployed.<br/>To deploy a multi-region cluster into those regional VPCs, ensure that the CIDRs of the VPCs do not overlap.<br/>If you intend to peer different VPCs to the same application VPC, ensure that the CIDRs of the VPCs do not overlap. |
```

File: docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-vpc-intro.md [hunk 2]
```diff
@@ -69,9 +69,9 @@ Each region in multi-region clusters must be deployed in a VPC. Depending on the
...
+| GCP Global | Create a single global VPC; each region of the cluster is deployed in the same VPC. |
+| GCP Custom | When creating the VPC, add a region for each region where you intend to deploy the cluster; each region of the cluster is deployed in the same VPC.<br/>If you plan to expand your cluster into new regions in the future, add those regions to the VPC when you create the VPC; _you can't expand into new regions after the VPC is created_. |
```
Comment (vitroth) [inline docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-vpc-intro.md:74]:
```suggestion
| GCP (Custom region selection) | When creating the VPC, you provide network blocks for each region where you intend to deploy the cluster; each region of the cluster is deployed in the same VPC.<br/>If you plan to expand your cluster into new regions in the future, add those regions to the VPC when you create the VPC; _you can not expand into new regions after the VPC is created_. |
```

File: docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-vpc-intro.md [hunk 3]
```diff
@@ -93,21 +93,34 @@ You can use the private IP addresses in the following ranges (per [RFC 1918](htt
...
+
+Addresses have the following additional restrictions:
+
+- VPC addresses can overlap with other VPCs, but not in the following circumstances:
+  - You want to peer the VPCs to the same application VPC. For example, if you have two different VPCs with overlapping addresses, you won't be able to peer them with the same application VPC.
 
-- Addresses can overlap with other VPCs, but not if they are peered to the same application VPC. YugabyteDB Managed warns you when you enter an overlapping range.
+  ![VPCs with overlapping CIDRs](/images/yb-cloud/managed-vpc-overlap-cidr.png)
+
+  - You want to use the VPCs for the same cluster. For example, if you have two VPCs in diffferent regions with overlapping addresses, you won't be able to use both for deploying a multi-region cluster.
```
Comment (mrajcevic01) [inline docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-vpc-intro.md:109]: Feel like this scenario will be run into more often, than the one listed above, and should be first in this list. Also check spelling on 'different'

File: docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-vpc-intro.md [hunk 4]
```diff
@@ -93,21 +93,34 @@ You can use the private IP addresses in the following ranges (per [RFC 1918](htt
...
+
+  ![VPCs with overlapping CIDRs in the same cluster](/images/yb-cloud/managed-vpc-overlap-cluster.png)
```
Comment (mrajcevic01) [inline docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-vpc-intro.md:111]: I think the image be indicating somehow that this is due to cross-cluster communication. Maybe have the two VPCs that are overlapping pointing to each other with the X,or the two regions with the overlap with the X?

**Review by lizayugabyte — APPROVED** [review]

**Pull request merged**